uint32 niro_can_message_number
uint32 msg_count
uint32 msg_status
float64 tosec
uint32 time
uint32 Left_Turn_Indicator
uint32 Left_Turn_Indicator_out_sign
uint32 Right_Turn_Indicator
uint32 Right_Turn_Indicator_out_sign
uint32 Fog_Light
uint32 Tail_Light
uint32 Head_Light
uint32 Up_Light
uint32 Auto_Light
uint32 Light_Status_out_sign
uint32 Emergency_Light
uint32 Emergency_Light_out_sign
uint32 Wiper_Operation
uint32 Wiper_Speed
uint32 Rear_Wiper_Operation
uint32 Rear_Wiper_Speed
uint32 LDWS_Switch
uint32 LDWS_Left_Warning
uint32 LDWS_Right_Warning
uint32 Brake_Switch
uint32 Gear_Position
uint32 Parking_Brake
uint32 Startup_key_state
uint32 Driver_Door_switch
uint32 Passenger_Door_switch
uint32 Rear_left_door_switch
uint32 Rear_right_door_switch
uint32 Eco_switch
uint32 Driver_Seat_Belt
uint32 Assistant_Seat_Belt
uint32 Rear_Side_Warning
uint32 Rear_Camera
uint32 L_SPAS_Warning
uint32 R_SPAS_Warning
uint32 Brake_Pedal_Pressure
float64 Throttle_Position
float64 N_Steering_Angle
uint32 N_Steering_Angle_velocity
uint32 N_Speed1
uint32 N_Speed2
float64 RPM
uint32 Lateral_Acceleration
uint32 Door_Lock
uint32 Air_Conditioner_Operation_Driver
uint32 Air_Conditioner_Operation_Assistant
uint32 Parking_Assistance_System
